src/kernel/arch/aarch64/tty.zig

```diff
@@ -95,6 +123,14 @@ fn writeChar(x: usize, y: usize, char: u8) void {
     }
 }
 
+///
+/// Write string to frame buffer using current text cursor.
+/// The current text cursor is updated.
+/// New lines ('\n) move to the next line.
+///
+/// Arguments:
+///     IN str: []const u8 - The string to draw
+///
 pub fn writeString(str: []const u8) !void {
```

Comment:
Looks like this function doesn't return an error, so can remove the `!`

Reply:
The ! is needed for interface conformance - added a comment to that effect.